css/index.css

```
@@ -253,3 +254,383 @@ body {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  background-color: white;
+  border: 1px solid black;
+  border-radius: 50%;
+  padding: 10px;
+  cursor: pointer;
+  width: 43px;
+  height: 45px;
+}
+
+.prev {
+  left: 10px;
+}
+
+.next {
+  right: 10px;
+}
+
+.indicators {
+  position: absolute;
+  bottom: 10px;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.dot {
+  height: 8px;
+  width: 16px;
+  margin: 0 5px;
+  background-color: #6C6B6A;
+  border-radius: 4px;
+  border: none;
+}
+
+.dot--active {
+  background-color: white;
+  width: 24px;
+  height: 8px;
+  border-radius: 4px;
+}
+
+.product__derecha {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap:25px;
+}
+
+.card {
+  position: relative;
+  width: 100%;
+  height: auto;
+  border-radius: 10px;
+  padding: 10px;
+  background-color: white;
+  box-shadow: 5px 5px 15px rgba(0, 0, 0, 0.05);
+  padding: 0px 52px 49px 12px;
+  color: #6C6B6A;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 22px;
+}
+
+.product__card-link{
+  font-size: 0;
+  inset: 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.card .product__categoria {
+  display: block;
+  color: #A6A6A6;
+  font-size: 12px; 
+  line-height: 16px;
+  opacity: 0.6;
+  font-weight: 600;
+}
+
+.product__nombre {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 22px;
+}
+
+.product__card-link:hover ~ .product__sales .product__nombre{
+  color: rgb(114, 164, 233);
+}
+
+.product__total {
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 25.14px;
+  color: #363535;
+}
+
+.product__valor {
+  display: flex;
+  justify-content: space-between;
+}
+
+.product__entrega {
+  color: #D14D41;
+  font-size: 12px;
+  font-style: italic;
+  font-weight: 400;
+  line-height: 15.08px;
+  text-align: left;
+  background-color: #FAEDEC; 
+  width: 83px;
+  height: 23.62px;
+  padding: 5px 4px 5px 4px;
+  border-radius: 4px 0px 0px 4px;
+}
+
+.product__card-link:hover ~ .product__sales .product__entrega{
+  color: #efac03;
+}
+
+
+.product__img{
+  display: flex;
+  justify-content: flex-end;
+}
+
+/* Desktop style */
+@media (min-width: 768px) {
+  .product {
+    grid-template-columns: 1fr 1fr;
+    max-width: 1343px;
+    height: auto;
+    padding: 50px 0px 0px 0px;
+  }
+  
+  .product__derecha {
+    grid-template-columns: 1fr 1fr;
+    margin-left: 39px;
+  }
+
+  .card {
+    width: 253px;
+    height: 265px;
+    padding: 0px 52px 49px 12px;
+  }
+
+  
+  .carousel__item {
+    min-width: 773px;
+    height: 563px;
+  }
+
+  .prev {
+    left: -20px;
+  }
+
+  .next {
+    right: -20px;
+  }
+
+  .indicators {
+    justify-content: flex-start;
+    margin-left: 268.65px;
+    margin-bottom: 151.1px;
+  }
+
+  .product__img{
+    display: flex;
+    margin-left: 3px;
+    width: 204px;
+    height: 117px;
+  }
+}
+
+/* Movil style */
+.carousel__categories {
+  max-width: 1350px;
+  margin: auto;
+  padding: 20px 0;
+}
+
+.categories {
+  position: relative;
+}
+
+h3 { 
```

Esto deberia usar una clase
```suggestion
.categories__title { 
```